services/121-service/src/seed-data/organization/organization-anonymous.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Looks like the deletion of the instance seed data slipped between the cracks

Reply:
I don't get it? Was this file deleted in main? @aberonni

Reply:
No, I mean that these seed files in `src/seed-data/organization` seem to be replacing the ones in `src/seed-data/instance` so I would assume that the latter folder would be deleted (but it doesn't seem that way).

Reply:
Ah good one!